frontend/src/types/ContentBlockTypes.ts

```diff
@@ -146,11 +147,9 @@ export class ContentTypeEnum {
   }
 }
 
-export const ColumnBlockValidChildren = [
-  ContentTypeEnum.BUTTON.id,
-  ContentTypeEnum.HEADING.id,
-  ContentTypeEnum.TEXT.id,
-  ContentTypeEnum.IMAGE.id,
+export const ColumnBlockInvalidChildren = [
+  ContentTypeEnum.AUDIO.id,
+  ContentTypeEnum.COLUMN.id,
   ContentTypeEnum.VIDEO.id,
```

Comment:
Is there a reason that audio and video are invalid column children?

Reply:
Audio doesn't really matter (because it isn't going to exist), but I can change it to allow videos.